src/controllers/auth/login.js

```diff
@@ -24,9 +24,10 @@ define(["apphost", "appSettings", "dom", "connectionManager", "loading", "layout
             page.querySelector("#txtManualPassword").value = "";
             loading.hide();
 
-            if (response.status === 401) {
+            if (response.status === 401 || response.status === 403) {
```

Comment:
```suggestion
            const UnauthorizedOrForbidden = [401, 403]
            if (UnauthorizedOrForbidden.inclues(response.status)) {
```

It doesn't make it shorter, but it's a bit more obvious and we can more the error codes to some utility module afterward.

For reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
Done 👍

Reply:
I think the other way was more readable for what it's worth.